core/utils/historied-data/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "historied-data"
```

Comment:
I'm not native speaker, but I think this should be renamed and also `substrate-` prefixed.

I think `substrate-historized-data` would be more correct.

Maybe @shawntabrizi can help us here? :)

Reply:
Neither sound right.

You want `historical-data`?

https://www.merriam-webster.com/dictionary/historical

Reply:
'historical' is sounding a bit funny to me (makes me think of chivalry and more generally historical events).

Maybe there is other possibilities :
here the purpose of the crate is to describe/manipulate a data that is stored with multiple representations depending on a given state (state being a point in history but could be something else for instance a blockchain fork plus a block number).

Reply:
`alternative-data`?

`parellel-data`?

I think `historical` is not really a word you should actively associate with physical history. When you clear your browser history, you would probably say you are clearing your historical data.